onebusaway-android/src/main/java/org/onebusaway/android/ui/TripPlanActivity.java

```diff
@@ -259,16 +258,6 @@ public void onBackPressed() {
         }
     }
 
-    @Override
```

Comment:
I would leave this in place - seems like it should work like this regardless of ES.